---
title: Teaching
bg: color5
color: white
icon: img/teaching.png
---

Part of our mission as a service core to the HMS community is to **provide bioinformatics training**. We are doing this through workshops, in-depth courses and individual consults.


### Workshop program 

Each month we run **one-day (Galaxy-based) [NGS workshops](http://scriptogr.am/ohofmann/about)** on RNA-Seq, ChIP-Seq, re-sequencing and other topics. These provide a basic understanding of the underlying technology and help with study designs and preliminary data analysis. No prior NGS or command line expertise is required. Our next courses are: 

* Introduction to RNA-Seq: November 17th
* Introduction to ChIP-Seq: December 2014
* More soon!

### In-depth NGS course

We want to ensure interested researchers can use [the same best practice methods and workflows](https://bcbio-nextgen.readthedocs.org/) we apply in a consult as well as utilize the available **cluster environment**, and **scale up** data analysis from Galaxy to a **command-line environment.** To that end HBC staff are developing an in-depth NGS workshop to be **launched in fall 2015** covering:

* Unix & High performance compute environments (Orchestra)
* Basic scripting (IPython, built around Software Carpentry)
* NGS analysis (command-line based, RNA-Seq, ChIP-Seq)
* Downstream analysis (RStudio)

Please subscribe to our **[teaching mailing list](TBA)** if you want us to notify you of new workshops and signup opportunities for the NGS course.

